gists/pull-request-guidelines.md

```diff
@@ -0,0 +1,69 @@
+# Pull Request (PR) Guidelines
+
+Guidelines for creating pull requests at LaunchPad Lab for three types of users: authors, code reviewers, and QA reviewers.
+
+- **Author**: The creator of the pull request.
+- **Code Reviewer**: A developer assigned to the pull request to review the new and/or modified code, not the application's functionality.
+- **QA Reviewer**: An individual assigned to the pull request to not only test the new and/or updated functionality introduced by the pull request, but also to raise any issues found related to existing functionality.
+
+## Authors
+- If there is an existing [pull request template](https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository), follow the steps and comments included
+  - These templates will come out-of-the-box when starting a new project using LPL's internal starting applications
+- Convert a PR to a [draft pull request](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/about-pull-requests#draft-pull-requests) if it is not ready for review yet (useful for [long-running](#long-running) prs)
+- Link to the appropriate Asana task(s) that the PR addresses
+- Specifically call out items / features that are out of scope
+- Provide screenshots and instructions to reproduce / find style issues if there are outstanding items that require design assistance
+- Add guidance about what the reviewer(s) should focus on
+- Document what the QA tester should be testing
```

Comment:
This dovetails into the QA/regression testing process we are trying to pilot in SF64 but in the future I'd love for these steps to be linked out to in another tool (ex. Asana) so they can be recalled easier in the future.

Reply:
I'm a little hesitant to link out to Asana, since this is our public Opex repo? Or were you intending for the other tool to also be exposed publicly?

Reply:
@chawes13 Sorry that was confusing. I just meant that this bullet point could reference a link out to Asana. It could read: `- Document what the QA tester should be testing (ideally these steps are captured in a PM tool (ex. Asana) and linked to in the PR)`. Considering how experimental this concept is, it probably doesn't make sense to include this now.

Reply:
😂😂😂😂😂😂😂😂 ohhhhhhh